### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by kRHYME7** [HyDE-Project/HyDE#1] [Announcements]
**Full Changelog**: https://github.com/HyDE-Project/HyDE/commits/v0.1.0

Freezing the old structure

<hr /><em>This discussion was created from the release <a href='https://github.com/HyDE-Project/HyDE/releases/tag/v0.1.0'>v0.1.0</a>.</em>

**Comment by 0xAbdiiii** [comment]
if you install this over the Prasanth's hyprdots, don't use the -r flag because there will be some oddities.

**Reply by 0xAbdiiii** [reply]
for example you will be have 2 waybars at the top because the hyprland.conf is now sourced from ~/.local/share/hyde and the -r makes it so that it doesn't overwrite the hyprland.conf in ~/.config/hypr so the new hyprland.conf will still be sourced but there will mostly be the same contents in files at both locations which also means you will be auto starting your apps 2 times after logging in.

**Reply by kRHYME7** [reply]
Thank you for having this here. 

Moving forward, this approach is better right?

**Reply by 0xAbdiiii** [reply]
> Thank you for having this here. 
> 

:)


> Moving forward, this approach is better right?


sorry for the late reply. i actually don't know how to feel about this change. at first thought, I'd want all my configurations in one place eg. ~/.config. but then i assume this change is made to make the dots better manageable so they don't mess each other up (hyprdots and user's dots)? in that case, this is a clever approach sourcing it from somewhere else and it does work really well too.

**Comment by rubiin** [comment]
I have made a custom bar which uses < 100MB and includes osd, notification , and soon. Its stable-ish but still needs some work for theme support (currently has catpuccin-mocha only). This is meant as a replacement of waybar and hyprpanel 

https://github.com/rubiin/FabricPanel

P.S I am open to more features

**Reply by rubiin** [reply]
![image](https://github.com/user-attachments/assets/c164d9aa-3c4a-4c8b-b206-a95375fcd41b)

using daily as of now

**Reply by kRHYME7** [reply]
Looking forward to this! given that python is widely supported across distros

**Reply by rubiin** [reply]
I also replicated swayosd . Its one to one replica of swaosd look

![image](https://github.com/user-attachments/assets/1c860f23-e991-4f97-95fd-242daf0f2666)

**Reply by rubiin** [reply]
https://github.com/user-attachments/assets/74e76ca0-9d66-442f-812f-6ca55ab95dcd

heres a small walkthrough

**Reply by rubiin** [reply]
> Looking forward to this! given that python is widely supported across distros

ags which uses gjs underneath was way limited and memory inefficient

**Reply by kRHYME7** [reply]
I still got input delays for swayosd on my end.


Can I use pipx for this? I want to install python dev stuff on my $user only. Saw that we can use venv so I guess yes?

**Reply by rubiin** [reply]
Use venv. Pipx uses a sandbox by default . Still i suggest the venv approach for now to keep deps upto date

**Reply by rubiin** [reply]
@kRHYME7 wlogout cloned 
![image](https://github.com/user-attachments/assets/a0b3e4c2-d333-45c8-85be-34bc253e570b)


I guess I will just clone the hyprdots style now so in future if hyde uses the panel

**Comment by rubiin** [comment]
@kRHYME7 do the volumecontrol and brightness control provide a way to turn off the notifications. I have another osd and dont like the notifications

**Reply by kRHYME7** [reply]
Will do it later.  Will you please open a Feat Req


I have some questions, do osd stuff doing something special like they send something in dbus? I want to know if there's a better way to check if any osd is already running or not.

**Reply by rubiin** [reply]
I think not. If you provide a way to quiet down the notifications , that would be enough

**Reply by kRHYME7** [reply]
do you have a daemon or anythig like a class soemthing to query ? or maybe a specific command to toggle the volume and brightness?

**Reply by rubiin** [reply]
you can check  `pid of hydepanel` i guess
